### DISCUSSION TITLE
Release notifications

### DISCUSSION THREAD

**Discussion by andy5995** [OpenRC/openrc#580] [Ideas]
@OpenRC, I think if you draft releases instead of pushing tags, then GitHub users who are watching releases on your repo will get email notifications (tags can be created when you draft a release).

**Comment by williamh** [comment]
Unless Github has changed something, I think github users can get notified when tags are pushed also right?

**Reply by andy5995** [reply]
If they're watching a repo it would should up in their feed, but they wouldn't receive email notifications, if I'm not mistaken.

https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/about-notifications

Doesn't explicitly say. I don't recall receiving email notifications for pushed tags from the various repos I subscribe to (I only subscribed to OpenRC *after* the 0.46 release).

**Comment by williamh** [comment]
I'll need to look to see if [gh](https://github.com/cli/cli) can manage releases.
I mostly work from the command line, so I'm not interested in using a github function that is only available via the web interface.

**Reply by andy5995** [reply]
> I'll need to look to see if [gh](https://github.com/cli/cli) can manage releases. I mostly work from the command line, so I'm not interested in using a github function that is only available via the web interface.

@williamh Releases can be created using a [release action](https://github.com/actions/create-release) in a workflow.

**Reply by vapier** [reply]
automatic releases based on tags sgtm

**Reply by eli-schwartz** [reply]
Or just

```
gh release create \
    "$tag" builddir/meson-dist/*.tar.xz{,.asc} \
    --verify-tag \
    --title="Release $tag" \
    --notes="In this release, foobar was added"
```
